workers/social/lib/social/models/session.coffee

```diff
@@ -57,6 +61,10 @@ module.exports = class JSession extends Model
           console.error err  if err?
 
 
+  @activeSession = secure (client, callback) ->
+    @one { clientId: client.sessionToken }, callback
```

Comment:
I don't have those information, just getting token from header. But using the `username` and `group` in returned `JSession` object.